packages/db/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import Knex from 'knex';
+
+export const dropAllTables = async ({
```

Comment:
I don't understand the motivation for creating a separate package here, much less one to just hold this method. This is only used from the drop all tables script, right?

I also worry about someone removing `private: true` and there suddenly being a package with a utility to drop all tables.

Reply:
> I don't understand the motivation for creating a separate package here, much less one to just hold this method. This is only used from the drop all tables script, right?

This is used in both `bin/db-local-drop.js` and `example/lambdas/db-drop-tables/src/index.ts`

Reply:
> I also worry about someone removing `private: true` and there suddenly being a package with a utility to drop all tables.

Don't do that, then? I don't think that we should be copying and pasting code around and not exporting it because of the fear that someone might then publish it and _then_ misuse it.

Reply:
Re: public/private --  given that change would have to pass code review *and* make it to release (and not be pulled) *and* be .... a poor choice implemented by another engineering team, that feels like more than sufficient to avoid badness.     If the goal is to be defensive re: that sort of thing, we should probably examine our setup and script the release such that it explicitly ignores paths or something along those line rather than avoid a package entirely.

Reply:
I think the chance of accidental publishing here is higher than we're estimating, but not something I'm going to hold this up on